MetamodelAlgemeen.md

```diff
@@ -298,7 +298,7 @@ Het wordt veelal als niet politiek correct beschouwd mensen als objecten te zien
 Een object heeft kenmerken waarvoor gegevens bekend zijn. 
 
 <aside class="definition">
-  <dfn>Gegeven</dfn>De betekenisvolle formulering van een waargenomen feit, waaraan een waarde kan worden toegekend.
+  <dfn>Gegeven</dfn>Een gegeven is een vastgelegde bewering over een getypeerde eigenschap van een object, met een waarde. 
```

Comment:
```suggestion
  <dfn>Gegeven</dfn>Een gegeven is een vastgelegde bewering over een getypeerde eigenschap van een object. 
```

"met een waarde" is onduidelijk. Een waarde van wat? Het is volgens mij ook niet nodig.

Reply:
Dat een gegeven een waarde heeft is juist essentieel. Dat kan er niet uit.